controllers/service.go

```diff
@@ -381,7 +392,7 @@ func isServiceTLSChanged(specAeroConf, statusAeroConf *asdbv1.AerospikeConfigSpe
 	specSVCPort := asdbv1.GetServicePort(specAeroConf)
 	statusSVCPort := asdbv1.GetServicePort(statusAeroConf)
 
-	return specTLSName != statusTLSName || specSVCPort != statusSVCPort
+	return specTLSName != statusTLSName || *specSVCPort != *statusSVCPort
```

Comment:
Won't this pointer de-reference throw panic when port is removed/added when TLS is already set.